djconnectwise/sync.py

```diff
@@ -1903,7 +1905,10 @@ def _assign_field_data(self, instance, json_data):
 
         instance.id = json_data['id']
         instance.name = json_data['name']
-        instance.description = json_data.get('description')
+        description = json_data.get('description')
+        if description is not None:
+            # '' string can be saved w/ is not None
```

Comment:
I don't understand why it's necessary to support saving an empty string. Is there some case I'm missing where it would be important to save ' '?

Reply:
Ah~ this is solution for solving unit test failures (None != ''). Any ideas for this?

Reply:
How about making sure the `json_data` that gets populated from the fixture has a notes field? Or this that already the case?

Reply:
Yeap, there are already. I will look into that a little more if we can do that other way.

Reply:
ok, thanks. 

I did a couple quick tests. It looks like Opportunity sync tests were the problem. I think it would be fine to simply make the notes on `API_SALES_OPPORTUNITY` set to `None`.

If there are no other issues, then can you change back all those `if notes is not None` back to `if notes` ?

Reply:
Thanks for your investigation. Fixture of opportunity is changed, and `is not None` is roll-backed.